### PR TITLE
Fixes #104 Replace jQuery.text function

### LINKED ISSUE #104: [jQuery] Replace jQuery.text function with native function
Currently we are using jQuery to update DOM contents but native browser functions exist to do this in a standard way so we don't need to rely on jQuery for this.

Example: 
https://github.com/Techtonica/keyboard-shortcuts-practice/blob/1a642f3ce4a136ec3b325beec781f4ab97b18935/public/scripts/main.js#L213

Search for all usages of the `.text(` function and replace with standard functionality.

[StackOverflow code example](https://stackoverflow.com/questions/1358810/how-do-i-change-the-text-of-a-span-element-using-javascript)

### PR DESCRIPTION
**Pull request by linhhvo**
Replaced all jQuery `.text( )` functions with vanilla JS `.textContent` as well as jQuery element selection `$( )` with `document.querySelector` in the same lines. I also replaced the ones commented out since I'm not sure if they will be used in the future. Let me know if I missed anything. Thanks!
Fixes #104

### CODE REVIEW

File: public/scripts/main.js
Comment (alodahl) [inline public/scripts/main.js:150]: 👍🏻

**Review by alodahl — APPROVED** [review]
Thank you!  

I'll wait for one more approval before merging.

**Review by vegetabill — APPROVED** [review]
Looks great! Thanks for replacing even the commented code. Very thorough job.

**Pull request merged**
